### ISSUE TITLE
[Module Proposal]: `avm-res-compute-availabilityset`

### ISSUE THREAD

**Issue by mbilalamjad** [Azure/Azure-Verified-Modules#656]
### Check for previous/existing GitHub issues/module proposals

- [X] I have checked for previous/existing GitHub issues/module proposals.

### Check this module doesn't already exist in the module indexes

- [X] I have checked for that this module doesn't already exist in the module indexes; or I'm proposing the module to be migrated from CARML/TFVM.

### Bicep or Terraform?

Terraform

### Module Classification?

Resource Module

### Module Name

avm-res-compute-availabilityset

### Module Details

Terraform equivalent of [Bicep Module](https://github.com/Azure/Azure-Verified-Modules/issues/509)

### Do you want to be the owner of this module?

No

**Comment by ASHR4** [comment]
I'm happy to take ownership of this module :)

**Comment by fafriha** [comment]
@mbilalamjad I can take this one as I'm the compute lead SME in our team.

**Comment by jtracey93** [comment]
Could @fafriha be the owner and work with @ASHR4 as a contributor, as FTEs must be owners today

**Comment by mbilalamjad** [comment]
Hi @fafriha,

Thanks for requesting/proposing to be an AVM module owner!

We just want to confirm **you agree to the below pages** that define what module ownership means:

- [Team Definitions & RACI](https://azure.github.io/Azure-Verified-Modules/specs/shared/team-definitions)
- [Shared Specification (Bicep & Terraform)](https://azure.github.io/Azure-Verified-Modules/specs/shared)
- [Module Support](https://azure.github.io/Azure-Verified-Modules/help-support/module-support)

Any questions or clarifications needed, let us know!

If you agree, please just **reply to this issue with the exact sentence below** (as this helps with our automation 👍):

"I CONFIRM I WISH TO OWN THIS AVM MODULE AND UNDERSTAND THE REQUIREMENTS AND DEFINITION OF A MODULE OWNER"

Thanks,

The AVM Core Team

#RR

**Comment by mbilalamjad** [comment]
> Could @fafriha be the owner and work with @ASHR4 as a contributor, as FTEs must be owners today

absolutely, let me get it assigned to @fafriha as the owner and then he could work with @ASHR4 on it together.

**Comment by prjelesi** [comment]
@fafriha can you please confirm based on message above?

**Comment by prjelesi** [comment]
@fafriha one more try - can you please confirm ownership?

**Comment by prjelesi** [comment]
Waiting for feedback from @mbilalamjad requested offline to confirm request.

@PmeshramPM for visibility

**Comment by mbilalamjad** [comment]
Just confirmed with the workload team that requested this and they will use the VMSS module instead over this so accordingly closing this proposal as no longer needed.
